cli/command/image/history_test.go

```diff
@@ -49,7 +48,6 @@ func notUTCTimezone() bool {
 }
 
 func TestNewHistoryCommandSuccess(t *testing.T) {
-	skip.If(t, notUTCTimezone, "expected output requires UTC timezone")
```

Comment:
Tried this locally, and looks like we may have the same issue as in the other test; the output is formatted for the local timezone, which may not be UTC, in which case the `.golden` files won't match;

```bash
=== RUN   TestNewHistoryCommandSuccess/non-human
    history_test.go:102: assertion failed:
        --- expected
        +++ actual
        @@ -1,3 +1,3 @@
        -IMAGE     CREATED AT             CREATED BY   SIZE      COMMENT
        -abcdef    2017-01-01T12:00:03Z   rose         0         new history item!
        +IMAGE     CREATED AT                  CREATED BY   SIZE      COMMENT
        +abcdef    2017-01-01T13:00:03+01:00   rose         0         new history item!

```


Not sure what the best solution is for this one though (don't think the `.golden` files allow for placeholders and/or fuzzy matching 🤔

Reply:
I'm running these tests in the dev env on macOS (`make dev`), which seems to set the timezone to UTC. Can we have tests run in UTC so they pass consistently?

(I'm also fine with just continuing to skip this test under non-UTC, but we'd still need to fix it so it _passes_ under UTC)

Reply:
Ah, yes, correct; running these tests inside the dev environment will run them inside a container (which default to use UTC). Running them directly on the host may not be the "common" way to run them, but (e.g.) to run a "one-off" test from within an IDE may be run that way.

I would suggest to either add back the skip for the whole test, or (now that we have subtests) to add a skip for that specific test (although I'd be fine with just adding back the `t.Skip` if we don't want to complicate things too much).

Reply:
So it turned out that even inside `make dev` these were being skipped, even though the dev env's timezone was UTC. 🤔 

I changed the semantics of `notUTCTimezone` to check for an offset != 0, which means these run now in the dev env.

https://go.dev/play/p/9o5SSnHUONR

Regardless, whether we skip the test case or not, I think the bigger issue is that it gets skipped in CI, so there's no safeguard against bugs creeping in for non-UTC users (statistically, most of them 🌎). So I'd like to see if we can make this unskipped in CI somehow at least.